### DISCUSSION TITLE
Stopping a discord bot, but not the program?

### DISCUSSION THREAD

**Discussion by lamaqdahodwala** [Rapptz/discord.py#6904] [Q&A] [answered]
I am trying to make a GUI for a discord bot, but for that to work, I need a way to stop the discord bot, but not the program itself. 
---
The method in particular would be kind of like the opposite of `bot.run`.
This method would allow us to take the bot offline, without stopping the program it is in. Does anyone know a method or solution for how to do this?

**Comment by Preocts** [comment, marked as answer]
Are you looking for `async close()`?

https://discordpy.readthedocs.io/en/stable/api.html#discord.Client.close

**Comment by lamaqdahodwala** [comment]
This is exactly what I was looking for, thank you :D
